tests/run-opengl.sh

```
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Run inside xvfb-run in CI. Offscreen also works with a local Mesa display.
+# The demo and UI suite create disposable profiles and media libraries.
+build_dir=$(realpath "${1:-build/release}")
+output_dir="$build_dir/opengl-renders"
+mkdir -p "$output_dir"
+export QT_QPA_PLATFORM=offscreen QT_QPA_PLATFORMTHEME= QT_IM_MODULE=compose
```

Shellcheck (0.11.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)